From 733a35fb6980994d16243cd8d52af892b81b610b Mon Sep 17 00:00:00 2001
From: Judith Mendez <jm@ti.com>
Date: Wed, 21 Feb 2024 18:33:16 -0600
Subject: [PATCH 14/27] arm64: dts: ti: k3-am625-sk: Remove mcasp0 node

Remove mcasp0 node in k3-am625-sk-rpi-hdr-ehrpwm overlay
since it is not enabled by default in the board DTS file.

Signed-off-by: Judith Mendez <jm@ti.com>
---
 arch/arm64/boot/dts/ti/k3-am625-sk-rpi-hdr-ehrpwm.dtso | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/ti/k3-am625-sk-rpi-hdr-ehrpwm.dtso b/arch/arm64/boot/dts/ti/k3-am625-sk-rpi-hdr-ehrpwm.dtso
index 48b3d71f1d79..f44616a0ed19 100644
--- a/arch/arm64/boot/dts/ti/k3-am625-sk-rpi-hdr-ehrpwm.dtso
+++ b/arch/arm64/boot/dts/ti/k3-am625-sk-rpi-hdr-ehrpwm.dtso
@@ -74,10 +74,6 @@
 	};
 };
 
-&mcasp0 {
-	status = "disabled";
-};
-
 &epwm0 {
 	status = "okay";
 	pinctrl-names = "default";
-- 
2.39.2

